client/src/components/Photo.jsx

```diff
@@ -2,10 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Photo = props => (
-  <div>
-    <li>
-      <img alt="" src={props.photo.url} />
-    </li>
+  <div className="d-flex flex-row flex-wrap justify-content-center">
+    <div className="d-flex flex-column">
+      <div className="thumbnail">
+        <a href={props.photo.url}>
+          <img className="img-responsive" alt="" src={props.photo.url} />
```

Comment:
Really small detail, but maybe specifying text in the alt attribute or removing it if unnecessary?